R/addins.R

```diff
@@ -1,25 +1,50 @@
-#' Style the active file
+
+#' Stylers for RStudio Addins
+#'
+#' Helper functions for styling via RStudio Addins.
+#'
+#' @section Auto-Save Option:
+#' By default, both of the RStudio Addins will apply styling to the (selected)
+#' file contents without saving changes. Automatic saving can be enabled by
+#' setting the environment variable \code{save_after_styling} to \code{TRUE}.
```

Comment:
We use markdown roxygen, so can you please change
```r
\code{...}
```
to 
```r
`...`
```
here and elsewhere in the PR?

Reply:
Oops! Sorry, old habits...